core/src/message/context.rs

```diff
@@ -135,6 +135,16 @@ pub struct ContextStore {
     initialized: BTreeSet<ProgramId>,
     awaken: BTreeSet<MessageId>,
     reply_sent: bool,
+    nonce: u64,
```

Comment:
```suggestion
    reservation_nonce: u64,
```

Here was previously nonce for outgoing messages so I'd rename this to avoid misunderstanding in future